examples/seismic/tutorials/Gradient_full_marmousi.py

```diff
@@ -0,0 +1,163 @@
+r'''
```

Comment:
nitpicking: filenames start with lower letters

more important: there's way too much duplicated code between this file and `gradient_snap_marmousi`. Can it not be made a single, self-contained example, with argparse used to allow the user to select among different modes

Reply:
Removed `Gradient_full_marmousi.py` and `Gradient_snap_marmousi.py` and added one script `fwi_gradient_marmousi.py` with argparse to choose between full gradient or snapshotting.